docs/src/main/sphinx/release/release-357.md

```diff
@@ -0,0 +1,72 @@
+# Release 357 (XX May 2021)
+
+## General
+
+* Add support for subquery expressions that return multiple columns. E.g., `SELECT x = (VALUES (1, 'a'))`. ({issue}`7773`, {issue}`7863`)
+* Allow aggregation pushdown when `COUNT(1)` is used with `GROUP BY`. ({issue}`7251`)
+* Add support for `CURRENT_CATALOG` and `CURRENT_SCHEMA`. ({issue}`7824`)
+* Add {func}`format_number` function. ({issue}`1878`)
+* Cast `row` to `json` with field names, instead of `array`. This behavior can be restored 
+  with the `deprecated.legacy-row-to-json-cast` configuration option. ({issue}`3536`)
+* Print dynamic filters summary in `EXPLAIN ANALYZE`. ({issue}`7874)
+* Minor performance improvement for queries using `IN` predicate with short list of constants. ({issue}`7840`)
+* Reduce memory usage for queries with `WINDOW` clause. ({issue}`7947`)
+* Fix incorrect handling of row expressions in `IN` predicate, quantified comparisons and scalar subqueries. ({issue}`7797`)
+* Fix failure when preparing a `GRANT` statement that contains quoted SQL keywords. ({issue}`7941`)
+
+## Security
+
+* Enforce materialized view creator security policies when view is fresh. ({issue}`7618`)
+
+## Web UI
+
+* Make the UI aware of principal-field (configured with `http-server.authentication.oauth2.principal-field`) when using `web-ui.authentication.type=ouath2`. ({issue}`7639`)
+
+## JDBC driver
+
+* Cancel Trino query execution when JDBC connection or statement is closed. ({issue}` 7819`)
+
+## BigQuery connector
+
+* Fix failures for queries accessing `information_schema.columns`. ({issue}`7830`)
+* Fix query failure when containing a predicate on BigQuery `string` type with single quotes. ({issue}`7784`)
+
+## ClickHouse connector
+
+* Improve performance of aggregation queries by computing aggregations within ClickHouse. Currently, the following aggregate functions are eligible for pushdown: count, min, max, sum and avg. ({issue}`7434`)
+
+## Elasticsearch connector
+
+* Add support for epoch millis exposed as String. ({issue}`7838`)
+* Retry requests with backoff when Elasticsearch is overloaded. ({issue}`7423`)
+
+## Kinesis connector
+
+* Allow configuring refresh interval to get table descriptions from S3 via the `kinesis.table-description-refresh-interval` config property. ({issue}`1609`)
+
+## MongoDB connector
+
+* Redact the value of `mongodb.credentials` in the server log. ({issue}`7862`)
+* Add support for dropping columns. ({issue}`7853`)
+
+## MySQL connector
+
+* Added JSON based identifier mapping that can be set with `case-insensitive-name-matching.config-file`. ({issue}`7434`)
+
+## Phoenix connector
+
+## Pinot connector
+                                
+* Fix filter parsing for passthrough queries. ({issue}`7161`)
```

Comment:
Any query with filter was broken? Or maybe we should be more specific here.

Reply:
@elonazoulay, do you have further details on this?

Reply:
Simple filtersworked before (i.e. column name = constant), but more complex expressions had issues. This is the list of what was fixed:
* Function calls
      * CASE statements
      * IN and NOT IN clauses
      * Comparision expressions
      * Logical expressions
      * Casts
      * Double literals

If that's too much it's probably sufficient just to say filter expressions though.

Reply:
This can be simplified to "Fix filter parsing for complex filter expressions in passthrough queries".